common-model/src/main/java/com/sequenceiq/common/api/telemetry/doc/TelemetryModelDescription.java

```diff
@@ -15,6 +15,7 @@
     public static final String TELEMETRY_REPORT_DEPLOYMENT_LOGS_ENABLED = "enable cluster deployment log reporting.";
     public static final String TELEMETRY_CLOUDWATCH_PARAMS = "telemetry - cloudwatch releated parameters";
     public static final String TELEMETRY_CLOUDWATCH_PARAMS_REGION = "telemetry - cloudwatch related AWS region (should be used only outside of AWS platform)";
```

Comment:
Minor: Although this belongs to the earlier #6851, `cloudwatch` should be spelled with the right casing: `CloudWatch`.